_rules/form-control-non-empty-accessible-name-e086e5.md

```diff
@@ -24,13 +24,6 @@ acknowledgments:
 
 This rule applies to any element that is [included in the accessibility tree](#included-in-the-accessibility-tree), and that has one of the following [semantic roles][]: `checkbox`, `combobox` (`select` elements), `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch`, `textbox`.
```

Comment:
I question why menuitemcheckbox and menuitemradio appear here and menuitem does not.

Reply:
Fair question. Similarly I think it's worth asking why this include spinbutton but not button. This is unrelated to the PR though, so I'll open a separate issue for it. Definitely something worth looking at.